samples/managed-identity/deploy-arm.yaml

```diff
@@ -0,0 +1,73 @@
+#this deploy yaml is for arm clusters so that managed identity can be kept separate for arm clusters.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mi-webapp
+  namespace: mi-webapp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mi-webapp
+  template:
+    metadata:
+      labels:
+        app: mi-webapp
+        # Label for pod identity
+        aadpodidbinding: mi-test-arm
+    spec:
+      containers:
+        - image: mindarodev.azurecr.io/samples/mi-webapp:v1
+          imagePullPolicy: IfNotPresent
+          name: mi-webapp
+          env:
+          # the below environment variable values come from the below sub and rg:
+          # Sub: Mindaro Testing (c2e0f009-a61a-4578-8a6d-5715ee782648)
+          # RG: testing-scenario
+          # Storage account name: mitestsa
+          # Managed identity name: mi-test
+          - name: STORAGE_ACCOUNT_NAME
```

Comment:
Are users expected to edit these and use their own subscriptions/applications to run these samples?

Reply:
@peterbom Yes correct , this is only for external dependencies like DB, managed identity, storage account. More documentation is [here](https://learn.microsoft.com/en-us/visualstudio/bridge/configure-bridge-to-kubernetes#local-configuration-using-kuberneteslocalprocessconfigyaml). This Yaml is particularly to test managed identity with b2k.

Reply:
@peterbom I misunderstood your question this is only for internal E2E testing, not used by any external users.

Reply:
Sounds good. The fact that it's in a `samples` folder gives the impression this is the recommended structure for customers, but I think it'd be better practice to avoid the duplication by using `ConfigMap` resources or deploying using Helm or Kustomize.

Given that this is just used internally, let's keep it as it is, and consider other approaches if it starts to look like a maintenance burden.